spec/requests/relationship_operations_spec.rb

```diff
@@ -279,7 +279,8 @@
   end
 
   # Polymorphic has-one relationship replacing
-  describe 'PATCH /tags/:id/relationships/taggable' do
+  # Polymorphic associations are broken: https://github.com/cerebris/jsonapi-resources/issues/1305
+  describe 'PATCH /tags/:id/relationships/taggable', pending: 'Broken upstream' do
```

Comment:
Looks like this:

* https://github.com/cerebris/jsonapi-resources/issues/1305

is a blocker for resolving this case, as I'm hitting the issue when I try to fix the polymorphic association support for JR 0.10. Damn.